### PR TITLE
Jenkins: setting advanced commit status

### PR DESCRIPTION
**Pull request by tjhei**

### CODE REVIEW

**Comment by tjhei** [comment]
@gassmoeller See how it posted the additional status? I think we can work with that! :tada:

**Comment by gassmoeller** [comment]
Very nice, now we only need to put some workers online. I will try to set up my laptop again some time next week.

**Comment by tjhei** [comment]
I think this setup might be useful to merge as is because it nicely displays the "trusted" and "astyle" status...

File: Jenkinsfile [hunk 1]
```diff
@@ -24,7 +24,8 @@ pipeline {
         echo "CHANGE_AUTHOR: ${env.CHANGE_AUTHOR}"
         echo "CHANGE_AUTHOR_DISPLAY_NAME: ${env.CHANGE_AUTHOR_DISPLAY_NAME}"
         echo "building on node ${env.NODE_NAME}"
-      }
+	githubNotify context: 'astyle', description: '', status: 'PENDING', targetUrl: ''
+	}
```
Comment (bangerth) [inline Jenkinsfile]: that's some funny indentation here :-)

File: Jenkinsfile [hunk 2]
```diff
@@ -43,21 +44,33 @@ pipeline {
...
         sh './doc/indent'
         sh 'git diff > changes-astyle.diff'
         archiveArtifacts artifacts: 'changes-astyle.diff', fingerprint: true
         sh '''
 	  git diff --exit-code || \
 	  { echo "Please check indentation, see artifacts in the top right corner!"; exit 1; }
 	  '''
-        }
+	  	githubNotify context: 'astyle', description: '', status: 'SUCCESS', targetUrl: ''
+	  }
```
Comment (bangerth) [inline Jenkinsfile]: same here

**Comment by bangerth** [comment]
I like this, but you are using some funny indentation in the jenkins file...

**Pull request closed without merging**